packager/packager.cc

```diff
@@ -250,6 +250,17 @@ class PackageInfoImpl final : public core::packages::PackageInfo {
         return sccID_;
     }
 
+    // PackageInfoImpl is the only implementation of PackageInfo
+    static PackageInfoImpl &from(core::GlobalState &gs, core::packages::MangledName pkg) {
+        ENFORCE(pkg.exists());
+        return from(*gs.packageDB().getPackageInfoNonConst(pkg));
+    }
+
+    const static PackageInfoImpl &from(const core::GlobalState &gs, core::packages::MangledName pkg) {
```

Comment:
```suggestion
    const static PackageInfoImpl &from(const core::GlobalState &gs, core::packages::MangledName pkg) const {
```

Reply:
Can't do that: this is a static helper method.

Reply:
`:facepalm:`